content/cfp.html

```diff
@@ -5,10 +5,12 @@
 callout_big_2: ...you
 ---
 
-<p>To submit a proposal to run a session at PyCon UK 2016, please complete <strong><a href="https://docs.google.com/forms/d/1IkIuN13wwDl1gH1L00HQ6UaX-3PWd3uFHuHvCmEJYGA/viewform">the proposal form</a></strong>.  Submissions are open until the end of June 28th (BST).</p>
-
 <p>This is a <strong>community conference</strong>. You’re coming to the event, and that makes <strong>you</strong> part of the community and makes this <strong>your</strong> conference. If you’ve done something that you think other people might be interested in, or that you think they should know about, <em>we would love to know about it</em>.</p>
 
+<a href="/tickets" class="btn small">Submit a proposal</a>
```

Comment:
Every browser will render this properly, even if it is deprecated, so invoking "practicality beats purity", I wouldn't worry about this.

I've got some time this evening though when I plan to get the [LESS files](https://github.com/PyconUK/2016.pyconuk.org/tree/less-files) incorporated into the code so that we can make CSS changes more easiliy.